docs/Accessibility.md

```diff
@@ -0,0 +1,75 @@
+---
+id: accessibility
+title: Accessibility
+layout: docs
+category: Guides
+permalink: docs/accessibility.html
+next: nativemodulesios
+---
+
+# Accessibility
+
+Accessibility on iOS encompasses many topics, but for many, accessibility is synonymous with VoiceOver, a technology available since iOS 3.0. It acts as a screen reader, allowing people with visual impairments to use their iOS devices. Click [here](https://developer.apple.com/accessibility/ios/) to learn more.
+
+## Making Accessible Apps
+
+### Coding Accessibly
+
+#### accessible
+
+When true, indicates that the view is an accessibility element. When a view is an accessibility element, it groups its children into a single selectable component. By default, all touchable elements are accessible.
```

Comment:
nit: I'd do `true` instead of true here

Also, I think it's not entirely clear what the implications of "it groups its children into a single selectable component" are for the developer - does this mean that an accessible component can only have a single child like Touchable components?